staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
We are still copying TLSConfig for mutation check. But copying with utilnet.CloneTLSConfig is taking more time and leading to error while using Go1.8. So instead of that we used simple assignment operation for copying.